src/content/blog/astro-480.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: "Astro 4.8"
+description: "Astro 4.8 is here! This release includes experimental support for Astro actions and rewriting, performance improvements, and more."
+publishDate: "May 9, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+  - sarah
+coverImage: "/src/content/blog/_images/astro-480/header_astro_4_8.webp"
+socialImage: "/src/content/blog/_images/astro-480/og_image_4_8.webp"
+lang: "en"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+
+<BlogSummary>
+Astro 4.8 is out now! This release includes experimental support for Astro actions and rewriting, performance improvements, and more.
+</BlogSummary>
+
+Full release highlights include:
+
+- [**Experimental: Astro Actions**](#experimental-astro-actions)
+- [**Experimental: Rewriting support**](#experimental-rewriting-support)
```

Comment:
What about *Request rewriting*? Just trying to think of a more-specific title here.